VideoLocker/src/main/java/org/edx/mobile/user/Account.java

```diff
@@ -0,0 +1,201 @@
+package org.edx.mobile.user;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class Account {
+    @SerializedName("username")
+    @NonNull
+    private String username;
+
+    @SerializedName("bio")
+    @Nullable
+    private Object bio;
+
+    @SerializedName("requires_parental_consent")
+    private boolean requiresParentalConsent;
+
+    @SerializedName("name")
+    @NonNull
+    private String name;
+
+    @SerializedName("country")
+    @Nullable
+    private String country; // ISO 3166 country code
+
+    @SerializedName("is_active")
+    private boolean isActive;
+
+    @SerializedName("profile_image")
+    @NonNull
+    private ProfileImage profileImage;
+
+    @SerializedName("year_of_birth")
+    @Nullable
+    private Integer yearOfBirth;
+
+    @SerializedName("level_of_education")
+    @Nullable
+    private String levelOfEducation;
+
+    @SerializedName("goals")
+    @Nullable
+    private String goals;
+
+    @SerializedName("language_proficiencies")
+    @NonNull
+    private List<LanguageProficiency> languageProficiencies = new ArrayList<LanguageProficiency>();
+
+    @SerializedName("gender")
+    @Nullable
+    private String gender;
+
+    @SerializedName("mailing_address")
+    @Nullable
+    private String mailingAddress;
+
+    @SerializedName("email")
+    private String email;
```

Comment:
I think email should probably be a mandatory field. We should get that
clarified from the API designers.

On Fri, Sep 25, 2015 at 8:16 PM, Brian Guertin <notifications@github.com>
wrote:

> In VideoLocker/src/main/java/org/edx/mobile/user/Account.java
> <https://github.com/edx/edx-app-android/pull/372#discussion_r40440551>:
>
> > +    private String goals;
> > +
> > +    @SerializedName("language_proficiencies")
> > +    @NonNull
> > +    private List<LanguageProficiency> languageProficiencies = new ArrayList<LanguageProficiency>();
> > +
> > +    @SerializedName("gender")
> > +    @Nullable
> > +    private String gender;
> > +
> > +    @SerializedName("mailing_address")
> > +    @Nullable
> > +    private String mailingAddress;
> > +
> > +    @SerializedName("email")
> > +    private String email;
>
> On second thought, I think weshould annotate unspecified nullness as
> @Nullable unless we know otherwise, just to play it safe.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-app-android/pull/372/files#r40440551>.
>